.github/workflows/cdk_release_code.yml

```diff
@@ -15,31 +15,26 @@ on:
       COMMIT_ID:
         required: true
         type: string
-      CREATE_INT_RELEASE_NOTES:
-        type: boolean
-        default: false
-      CREATE_INT_RC_RELEASE_NOTES:
-        type: boolean
-        default: false
-      CREATE_PROD_RELEASE_NOTES:
-        type: boolean
-        default: false
-      MARK_JIRA_RELEASED:
-        type: boolean
-        default: false
-    secrets:
-      CLOUD_FORMATION_DEPLOY_ROLE:
+      CDK_APP_NAME:
         required: true
-      CDK_PULL_IMAGE_ROLE:
+        type: string
+      LOG_RETENTION_IN_DAYS:
         required: true
-      DEV_CLOUD_FORMATION_CHECK_VERSION_ROLE:
-        required: false
-      INT_CLOUD_FORMATION_CHECK_VERSION_ROLE:
+        type: string
+      GET_IMPORT_COMMANDS:
         required: false
-      PROD_CLOUD_FORMATION_CHECK_VERSION_ROLE:
+        type: string
+      JQ_VARIABLES:
         required: false
-      DEV_CLOUD_FORMATION_EXECUTE_LAMBDA_ROLE:
+        type: string
+      JQ_REPLACEMENTS:
```

Comment:
you are not using the vars
GET_IMPORT_COMMANDS
JQ_VARIABLES
JQ_REPLACEMENTS
as you are getting the values every time